.circleci/workflows.yml

```diff
@@ -259,28 +259,36 @@ commands:
   publish_rpms:
     description: Runs the commands to build n publish RPM Packages for Gravitee.io AM
     parameters:
+      dry_run:
+        type: boolean
+        default: true
+        description: "Run in dry run mode?"
       graviteeio_version:
```

Comment:
We should manage the parameter `rc_requested` to ignore publish_rpm for RC/alpha releases